README.md

```diff
@@ -80,24 +79,181 @@ After you input your application credentials, you can click the setup button, wh
 
 **Important**: Make sure your Resources/MobiledgeXSettings.asset file has the correct information for your application. 
 
-Once that setup has been completed, you can very easily call all the necessary API requests to connect to a cloudlet with your application deployed. Here is some example code using the MobiledgeXIntegration class that comes with the package : 
+Once that setup has been completed, you can very easily call all the necessary API requests to connect to a cloudlet with your application deployed. Here is some example code using the MobiledgeXIntegration class that comes with the package 
+
+
+**Getting Edge Connection Url**
+
+
 
 ```csharp
 using MobiledgeX;
+using DistributedMatchEngine;
+using UnityEngine;
+using System.Collections;
+
+[RequireComponent(typeof(MobiledgeX.LocationService))]
+public class NetworkManager : MonoBehaviour
+{ 
+    IEnumerator Start()
+    {
+        yield return StartCoroutine(MobiledgeX.LocationService.EnsureLocation()); // Location is needed to connect you to the closet edge
+        GetEdgeConnection();
+    }
+     async void GetEdgeConnection()
+    {
+        MobiledgeXIntegration mxi = new MobiledgeXIntegration();
+        try
+        {
+            await mxi.RegisterAndFindCloudlet();
+        }
+        catch(DmeDnsException)
+        {
+            mxi.UseWifiOnly(true); // if you carrier is not supported yet, WifiOnly will connect you to the closet MobiledgeX Public Cloud
+            await mxi.RegisterAndFindCloudlet();
+        }
+
+        mxi.GetAppPort(LProto.L_PROTO_HTTP); // Get the port of the desired protocol
+        string url = mxi.GetUrl("http"); // Get the url of the desired protocol
+    }
+    
+}
+```
+
+
+MobiledgeX requires Location and Mobile phone carrier information (specifically MccMnc code)
```

Comment:
We dont require phone info as a developer can use UseWifiOnly

Reply:
MobiledgeX uses Location and the phone carrier [MCCMNC](https://developers.mobiledgex.com/support)

Is that better ?

Reply:
hmm, maybe this more like the sentence is in the wrong location since after code, I would expect an explanation of the code or just moving to the next section